training/model_management/src/azureml/model/mgmt/processors/transformers/whisper/predict.py

```diff
@@ -233,5 +227,7 @@ def predict(
         predicted_ids = model.generate(input_features, forced_decoder_ids=forced_decoder_ids,
                                        max_new_tokens=max_new_tokens)
         transcription = tokenizer.batch_decode(predicted_ids, skip_special_tokens=True)[0]
+
         result.append({"text": transcription})
-    return result
+
+    return pd.DataFrame(result)
```

Comment:
Please update `predict` docs for return type

Reply:
Done